framework/test/unit/node/forger/forger.spec.ts

```diff
@@ -126,12 +126,21 @@ describe('forger', () => {
 
 	describe('Forger', () => {
 		describe('updateForgingStatus', () => {
+			beforeEach(() => {
```

Comment:
it should have the tests for cases below?
- when force false and not matched maxHeightPreviouslyForged
- when force false and matched maxHeightPreviouslyForged
- when force false and data does not exist
- when force true